src/handle/command.ml

```diff
@@ -407,31 +407,79 @@ let handle : (Path.t -> Sign.t) -> sig_state -> p_command ->
             (* Just ignore the command with a warning. *)
             wrn pe.pos "Proof aborted."; ss
         | _ ->
-            (* Check that no metavariable remains. *)
-            if LibTerm.Meta.has true a then
-              (fatal_msg "The type of %a has unsolved metavariables.\n"
-                 pp_uid id;
-               fatal pe.pos "We have %a : %a." pp_uid id pp_term a);
-            (match t with
-             | Some(t) when LibTerm.Meta.has true t ->
-                 fatal_msg
-                   "The definition of %a has unsolved metavariables.\n"
-                   pp_uid id;
-                 fatal pe.pos "We have %a : %a ≔ %a."
-                   pp_uid id pp_term a pp_term t
-             | _ -> ());
```

Comment:
This chunk of code has just been moved to the case `P_proof_end`.

Reply:
Yes, but the remark is still valid.